lib/sisimai/bite/email/zoho.rb

```diff
@@ -0,0 +1,167 @@
+module Sisimai::Bite::Email
+  module Zoho
+    # Sisimai::Bite::Email::Zoho parses a bounce email which created by Zoho Mail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Zoho.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'     => %r/mailer-daemon[@]mail[.]zoho[.]com\z/,
+        :'subject'  => %r{\A(?:
+             Undelivered[ ]Mail[ ]Returned[ ]to[ ]Sender
+            |Mail[ ]Delivery[ ]Status[ ]Notification
+            )
+        }x,
+        :'x-mailer' => %r/\AZoho Mail\z/,
+      }
+      Re1 = {
+        :begin  => %r/\AThis message was created automatically by mail delivery/,
+        :rfc822 => %r/\AReceived:[ \t]*from mail[.]zoho[.]com/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        expired: %r/Host not reachable/
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Zoho Mail: https://www.zoho.com'; end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->